### PR TITLE
add Stride

### PR DESCRIPTION
**Pull request by riley-stride**
Add stride

### CODE REVIEW

File: stride/assetlist.json [hunk 1]
```diff
@@ -0,0 +1,114 @@
...
+        ],
+        "base": "ustatom",
+        "name": "stATOM",
+        "display": "statom",
+        "symbol": "stATOM",
+        "logo_URIs": {
+          "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/stride/images/statom.png",
+          "svg": "https://raw.githubusercontent.com/cosmos/chain-registry/master/stride/images/statom.svg"
+        },
+        "coingecko_id": "statom"
```
Comment (JeremyParish69) [inline stride/assetlist.json:69]: not a CGID in https://api.coingecko.com/api/v3/coins/list
is it supposed to be "stride-staked-atom"?

File: stride/assetlist.json [hunk 2]
```diff
@@ -0,0 +1,114 @@
...
+        ],
+        "base": "ustosmo",
+        "name": "stOSMO",
+        "display": "stosmo",
+        "symbol": "stOSMO",
+        "logo_URIs": {
+          "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/stride/images/stosmo.png",
+          "svg": "https://raw.githubusercontent.com/cosmos/chain-registry/master/stride/images/stosmo.svg"
+        },
+        "coingecko_id": "stosmo"
```
Comment (JeremyParish69) [inline stride/assetlist.json:90]: doesn't exist

File: stride/assetlist.json [hunk 3]
```diff
@@ -0,0 +1,114 @@
...
+        ],
+        "base": "ust",
+        "name": "Stride",
+        "display": "st",
+        "symbol": "ST",
+        "logo_URIs": {
+          "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/stride/images/st.png",
+          "svg": "https://raw.githubusercontent.com/cosmos/chain-registry/master/stride/images/st.svg"
+        },
+        "coingecko_id": "stride"
```
Comment (JeremyParish69) [inline stride/assetlist.json:27]: doesn't exist please remove until added to CoinGecko

File: stride/chain.json [hunk 4]
```diff
@@ -0,0 +1,114 @@
+{
+    "$schema": "../chain.schema.json",
+    "chain_name": "stride",
+    "status": "live",
+    "network_type": "mainnet",
+    "updatelink": "https://raw.githubusercontent.com/osmosis-labs/osmosis/main/chain.schema.json",
+    "pretty_name": "Stride",
+    "chain_id": "stride-1",
+    "bech32_prefix": "st",
```
Comment (JeremyParish69) [inline stride/chain.json:9]: the PR on slip173 is for prefix 'stride'. Is this 'st' meant to be 'stride'?

File: stride/chain.json [hunk 5]
```diff
@@ -0,0 +1,114 @@
...
+            {
+                "address": "grpc-osmosis-ia.notional.ventures:443",
+                "provider": "Notional"
+            }
+        ]
+    },
+    "explorers": [
+        {
+            "kind": "mintscan",
+            "url": "https://www.mintscan.io/osmosis",
```
Comment (JeremyParish69) [inline stride/chain.json:101]: no explorers yet?

File: stride/assetlist.json (hunk 2 above)
Comment (riley-stride) [reply stride/assetlist.json:90]: replaced with stride-staked-atom

File: stride/chain.json (hunk 5 above)
Comment (riley-stride) [reply stride/chain.json:101]: updated to testnet explorer

File: stride/chain.json (hunk 4 above)
Comment (riley-stride) [reply stride/chain.json]: yes, fixed. thanks!

File: stride/chain.json [hunk 6]
```diff
@@ -0,0 +1,101 @@
...
+    "chain_name": "stride",
+    "status": "live",
+    "network_type": "mainnet",
+    "updatelink": "https://raw.githubusercontent.com/osmosis-labs/osmosis/main/chain.schema.json",
+    "pretty_name": "Stride",
+    "chain_id": "stride-1",
+    "bech32_prefix": "stride",
+    "daemon_name": "strided",
+    "node_home": "$HOME/.strided",
+    "genesis": {
```
Comment (JeremyParish69) [inline stride/chain.json:12]: osmosis genesis? needs correcting

File: stride/chain.json [hunk 7]
```diff
@@ -0,0 +1,101 @@
+{
+    "$schema": "../chain.schema.json",
+    "chain_name": "stride",
+    "status": "live",
+    "network_type": "mainnet",
+    "updatelink": "https://raw.githubusercontent.com/osmosis-labs/osmosis/main/chain.schema.json",
```
Comment (JeremyParish69) [inline stride/chain.json:6]: remove this

File: stride/chain.json [hunk 8]
```diff
@@ -0,0 +1,101 @@
...
+                "average_gas_price": 0.025,
+                "high_gas_price": 0.04
+            }
+        ]
+    },
+    "codebase": {
+        "git_repo": "https://github.com/Stride-Labs/stride",
+        "recommended_version": "v1.0.0",
+        "binaries": {
+            "linux/amd64": "https://github.com/osmosis-labs/osmosis/releases/download/v10.0.0/osmosis-10.0.0-linux-amd64",
```
Comment (JeremyParish69) [inline stride/chain.json:34]: shouldn't have osmosis binaries

File: stride/chain.json [hunk 9]
```diff
@@ -0,0 +1,101 @@
...
+    "peers": {
+        "seeds": [
+            {
+                "id": "83adaa38d1c15450056050fd4c9763fcc7e02e2c",
+                "address": "ec2-44-234-84-104.us-west-2.compute.amazonaws.com:26656",
+                "provider": "notional"
+            },
+            {
+                "id": "7c66126b64cd66bafd9ccfc721f068df451d31a3",
+                "address": "osmosis-seed.sunshinevalidation.io:9393",
```
Comment (JeremyParish69) [inline stride/chain.json:49]: still have osmosis apis

File: stride/assetlist.json [hunk 10]
```diff
@@ -0,0 +1,93 @@
...
+        "logo_URIs": {
+          "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/stride/images/strd.png",
+          "svg": "https://raw.githubusercontent.com/cosmos/chain-registry/master/stride/images/strd.svg"
+        },
+        "coingecko_id": ""
+      },
+      {
+        "denom_units": [
+          {
+            "denom": "ustatom",
```
Comment (asalzmann) [inline stride/assetlist.json:32]: stuatom @riley-stride

File: stride/assetlist.json [hunk 11]
```diff
@@ -0,0 +1,93 @@
...
+            "exponent": 0
+          },
+          {
+            "denom": "statom",
+            "exponent": 6
+          }
+        ],
+        "base": "ustatom",
```
Comment (asalzmann) [inline stride/assetlist.json:40]: stuatom

File: stride/assetlist.json [hunk 12]
```diff
@@ -0,0 +1,93 @@
...
+        "logo_URIs": {
+          "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/stride/images/statom.png",
+          "svg": "https://raw.githubusercontent.com/cosmos/chain-registry/master/stride/images/statom.svg"
+        },
+        "coingecko_id": "stride-staked-atom"
+      },
+      {
+        "denom_units": [
+          {
+            "denom": "ustosmo",
```
Comment (asalzmann) [inline stride/assetlist.json:53]: stuosmo

File: stride/assetlist.json [hunk 13]
```diff
@@ -0,0 +1,93 @@
...
+            "exponent": 0
+          },
+          {
+            "denom": "stosmo",
+            "exponent": 6
+          }
+        ],
+        "base": "ustosmo",
```
Comment (asalzmann) [inline stride/assetlist.json:61]: stuosmo

File: stride/assetlist.json [hunk 14]
```diff
@@ -0,0 +1,93 @@
...
+          {
+            "denom": "ustjuno",
+            "exponent": 0
+          },
+          {
+            "denom": "stjuno",
+            "exponent": 6
+          }
+        ],
+        "base": "ustjuno",
```
Comment (asalzmann) [inline stride/assetlist.json:82]: stujuno

File: stride/assetlist.json [hunk 15]
```diff
@@ -0,0 +1,93 @@
...
+        "logo_URIs": {
+          "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/stride/images/stosmo.png",
+          "svg": "https://raw.githubusercontent.com/cosmos/chain-registry/master/stride/images/stosmo.svg"
+        },
+        "coingecko_id": ""
+      },
+      {
+        "denom_units": [
+          {
+            "denom": "ustjuno",
```
Comment (asalzmann) [inline stride/assetlist.json:74]: stujuno

Comment (asalzmann) [reply stride/assetlist.json:74]: is this the juno denom?

File: stride/chain.json [hunk 16]
```diff
@@ -0,0 +1,135 @@
...
+    "pretty_name": "Stride",
+    "chain_id": "stride-1",
+    "bech32_prefix": "stride",
+    "daemon_name": "strided",
+    "node_home": "$HOME/.strided",
+    "genesis": {
+        "genesis_url": "https://raw.githubusercontent.com/Stride-Labs/testnet/infra-test/poolparty/infra/genesis.json"
+    },
+    "key_algos": [
+        "secp256k1"
```
Comment (asalzmann) [inline stride/chain.json:15]: is this the right algo? I thought we were using ed25519

File: stride/chain.json [hunk 17]
```diff
@@ -0,0 +1,135 @@
...
+    ],
+    "slip44": 118,
+    "fees": {
+        "fee_tokens": [
+            {
+                "denom": "ustrd",
+                "fixed_min_gas_price": 0,
+                "low_gas_price": 0,
+                "average_gas_price": 0.025,
```
Comment (asalzmann) [inline stride/chain.json:24]: should we set this to 0? this is the default option in keplr

File: stride/chain.json (hunk 6 above)
Comment (riley-stride) [reply stride/chain.json:11]: done

File: stride/chain.json (hunk 7 above)
Comment (riley-stride) [reply stride/chain.json]: done

File: stride/chain.json (hunk 8 above)
Comment (riley-stride) [reply stride/chain.json]: done

File: stride/chain.json (hunk 9 above)
Comment (riley-stride) [reply stride/chain.json]: replaced with stride apis

File: stride/assetlist.json (hunk 10 above)
Comment (riley-stride) [reply stride/assetlist.json]: done

File: stride/assetlist.json (hunk 11 above)
Comment (riley-stride) [reply stride/assetlist.json]: done

File: stride/assetlist.json (hunk 12 above)
Comment (riley-stride) [reply stride/assetlist.json]: done

File: stride/assetlist.json (hunk 13 above)
Comment (riley-stride) [reply stride/assetlist.json]: done

File: stride/assetlist.json (hunk 14 above)
Comment (riley-stride) [reply stride/assetlist.json]: done

File: stride/assetlist.json (hunk 15 above)
Comment (riley-stride) [reply stride/assetlist.json]: done, yes

File: stride/chain.json (hunk 16 above)
Comment (riley-stride) [reply stride/chain.json]: good catch, replaced with ed25519

File: stride/chain.json (hunk 17 above)
Comment (riley-stride) [reply stride/chain.json]: yes, set to 0

**Pull request merged**